src/BinSkim.Rules/PERules/BA2015.EnableHighEntropyVirtualAddresses.cs

```diff
@@ -60,8 +60,16 @@ public override AnalysisApplicability CanAnalyzePE(PEBinary target, Sarif.Proper
             if (portableExecutable.IsKernelMode) { return result; }
 
             reasonForNotAnalyzing = MetadataConditions.ImageIsNot64BitBinary;
-            if (portableExecutable.PEHeaders.PEHeader.Magic != PEMagic.PE32Plus) { return result; }
-
+            if (portableExecutable.PEHeaders.PEHeader.Magic != PEMagic.PE32Plus
```

Comment:
>portableExecutable.PEHeaders.PEHeader.Magic != PEMagic.PE32Plus [](start = 16, length = 63)

complex condition, let's comment what this logic is accomplishing, flagging an AnyCpu binary that is configured such that it might run as 64-bit